scapy/arch/windows/__init__.py

```diff
@@ -177,24 +177,40 @@ def _vbs_exec_code(code, split_tag="@"):
             yield l
     os.unlink(tmpfile.name)
 
-def _vbs_get_hardware_iface_guid(devid):
+def _get_hardware_iface_guid(devid):
```

Comment:
Could you add a docstring?

Reply:
Rather no :(.
Sorry.. Due to my "english" and because I am not familiar with scapy and net domain at all/only a bit :(.
Could you help me and add it?

Reply:
Looks good to me. Please replace 'Gets' with 'Get' and 'exists' with 'exist'. In doubt, you can use Google Docs to perform spellchecks =)